src/lib.rs

```diff
@@ -5,7 +5,7 @@
   future_incompatible,
   nonstandard_style,
   rust_2018_idioms,
-  missing_docs
+  // missing_docs
```

Comment:
I'd re-enable this and add all the required comments before merging.

Reply:
Agree.